descriptions/api.github.com/api.github.com.json

```
@@ -29864,6 +29864,12 @@
         }
       }
     },
+    "/repos/{owner}/{repo}/hooks/{hook_id}/deliveries": {
+    },
+    "/repos/{owner}/{repo}/hooks/{hook_id}/deliveries/{delivery_id}": {
+    },
+    "/repos/{owner}/{repo}/hooks/{hook_id}/deliveries/{delivery_id}/attempts": {
+    },
```

I don't think empty path objects should be added to the OpenAPI spec. Could you maybe add a validation rule to prevent that from happening? That'd be great
```suggestion
    "/repos/{owner}/{repo}/hooks/{hook_id}/deliveries": {
    },
    "/repos/{owner}/{repo}/hooks/{hook_id}/deliveries/{delivery_id}": {
    },
    "/repos/{owner}/{repo}/hooks/{hook_id}/deliveries/{delivery_id}/attempts": {
    },
```